test/pdfboxing/common_test.clj

```diff
@@ -10,18 +12,28 @@
 
 (deftest obtain-document-returns-pddocument-if-provided-string-check
   (testing "obtain-document returns a PDDocument if a string path to a PDF is a supplied"
-    (is (true?  (instance?
-                   org.apache.pdfbox.pdmodel.PDDocument
-                   (obtain-document "test/pdfs/hello.pdf"))))))
+    (let [doc (obtain-document "test/pdfs/hello.pdf")]
```

Comment:
Looking at this change, and the three below, could this be done via `with-open` like you've used it in **split.clj**. Something like:

```clojure
(with-open [doc (common/obtain-document "test/pdfs/hello.pdf")]
  (is (true? (instance? PDDocument doc))))
```
Instead of the `try`/`finally`?